management/commands/wait_for_db.py

```
@@ -0,0 +1,22 @@
+from time import sleep
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    help = "Wait for database to be available"
+
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_conn = None
+        attempts = 0
+        while not db_conn:
+            try:
+                connections['default'].cursor()
+                db_conn = True
+            except OperationalError:
+                attempts += 1
+                self.stdout.write(f"DB not ready, retry {attempts}...")
+                sleep(1)
```

The while loop retries indefinitely with a fixed 1 second delay. Consider making the delay configurable (via an environment variable) or implementing an exponential backoff to avoid very frequent retries if Postgres is slow to initialize. Also consider adding an optional maximum retry count or timeout so the process can fail fast on misconfiguration instead of hanging forever. This will make container behavior clearer when DB credentials/host are wrong.